sdk/storage/Azure.ResourceManager.Storage/api/Azure.ResourceManager.Storage.netstandard2.0.cs

```diff
@@ -1126,6 +1130,23 @@ public partial class EncryptionScopeKeyVaultProperties
         public override string ToString() { throw null; }
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct FailoverType : System.IEquatable<Azure.ResourceManager.Storage.Models.FailoverType>
```

Comment:
Can we rename this to `StorageAccountFailoverType`?

Reply:
Done